### ISSUE TITLE
Training script of Reward model

### ISSUE THREAD

**Issue by OrangeSodahub** [WM-PO/WMPO#5]
Hi, we want to retrain your reward model (VideoMAE) on new data, but we couldn't find any training scripts for it. Could you provide them?

**Comment by OrangeSodahub** [comment]
@fangqi-Zhu Also is it possible to provide the base policy finetuning script (e.g., openvla-oft)?

**Comment by fangqi-Zhu** [comment]
Sorry for the late reply. I will update all the scripts you need later today. If you have any additional requirements afterward, please feel free to email me, and I will share my WeChat with you. Thank you for your interest in our work.

**Comment by fangqi-Zhu** [comment]
We have uploaded the reward model code and the base policy fine-tuning scripts, for example `dependencies/openvla-oft/train_coffee.sh`. The SFT data has also been uploaded to Hugging Face. If you have any other questions or issues, please let me know.
